docs/source/getting-started/institutional-faq.md

```diff
@@ -0,0 +1,246 @@
+broken down by their roles within organizations.
+
+# For all
+
+## Is appropriate for adoption within a larger institutional context?
+
+Yes! JupyterHub has been used at-scale for large pools of users, as well
+as complex and high-performance computing. For example, UC Berkeley uses
+JupyterHub for its Data Science Education Program courses (serving over
+3,000 students). The Pangeo project uses JupyterHub to provide access
+to scalable cloud computing with Dask. JupyterHub is stable customizable
+to the use-cases of large organizations.
+
+## I keep hearing about Jupyter Notebook, JupyterLab, and now JupyterHub. What’s the difference?
+
+Here is a quick breakdown of these three tools:
+
+* **The Jupyter Notebook** is a document specification (the `.ipynb`) file that interweaves
+  narrative text with code cells and their outputs. It is also a graphical interface
+  that allows users to edit these documents
+* **JupyterLab** is a flexible and extendible user interface for interactive computing. It
+  has several extensions that are tailored for using Jupyter Notebooks, as well as extensions
+  for other parts of the data science stack.
+* **JupyterHub** is an application that can manage **multiple users** with interactive computing
+  sessions, as well as connect with infrastructure those users wish to access. It can provide
+  remote access to Jupyter Notebooks and Jupyter Lab for many people, and can connect them with
+  other compute infrastructure.
+
+# For management
+
+## Briefly, what problem does JupyterHub solve for us?
+
+JupyterHub provides a shared platform for data science and collaboration.
+It allows users to utilize familiar data science workflows (such as the scientific python stack,
+the R tidyverse, and Jupyter Notebooks) on institutional infrastructure. It also allows administrators
+some control over access to resources, security, authentication, and user identity.
+
+## Is JupyterHub mature? Why should we trust it?
+
+Yes - the core JupyterHub application recently
+reached 1.0 status, and is considered stable and performant for most institutions.
+JupyterHub has also been deployed (along with other tools) to work on
+scalable infrastructure, large datasets, and high-performance computing.
+
+## Who else uses JupyterHub?
+
+JupyterHub is used at a variety of institutions in academia,
+industry, and governmental research labs. It is most-commonly used by two kinds of groups:
+
+* Small teams (e.g., data science teams, research labs, or collaborative projects) to provide a
+  shared resource for interactive computing, collaboration, and analytics.
+* Large teams (e.g., a department, a large class, or a large group of remote users) to provide
+  access to organizational hardware, data, and analytics environments at scale.
+
+## How does JupyterHub compare with hosted products, like Google Colaboratory, RStudio.cloud, or Anaconda Enterprise?
+
+Like the tools listed above, JupyterHub provides access to interactive computing
```

Comment:
I would not start with "It is like the others..." and instead lead with "JupyterHub puts you in control of your data and infrastructure as well as being vendor neutral (reduced lock in)." The first sentence should be three good advantages (potentially taking advantage of peoples worries about these other products) of JupyterHub.

Reply:
Is this not resolved for you?

Reply:
Ah, didn't see it had changed. Was expecting you to mark it as resolved or say something like "I disagree..". Looks good!